stage1/units/units/sysusers.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Create /etc/passwd and /etc/group
+DefaultDependencies=false
+OnFailureJobMode=fail
+
+[Service]
+Type=oneshot
+Restart=no
+Environment="SYSTEMD_LOG_LEVEL=err"
```

Comment:
It prints the user and groups generated by `systemd-sysusers`, I don't think that's really useful. But now that you mention it, it should probably be `SYSTEMD_LOG_LEVEL=warn`, I'll try it.